packages/stateless/src/lib/provider/channel-data.type.ts

```diff
@@ -0,0 +1,50 @@
+export type ChannelData = SlackChannelData | SlackUserData | WebhookData | PhoneData;
```

Comment:
This is basically interface for stateless since it doesn't have access to shared we unfortunately can't reuse, but its fine cause they don't necessarily need to stay the same in the future and also this requires just subset of the ChannelAddress + ChannelConnection data